src/components/app/content/cardList/card/index.js

```diff
@@ -0,0 +1,54 @@
+import React, {useCallback, useEffect, useState} from 'react';
+import './index.css'
+import CardHeader from "./cardHeader";
+import CardBody from "./cardBody";
+
+function Card(props) {
+    const cardData = props.cardData;
+    const cardId = cardData.id;
+
+    const [tempState, setTempState] = useState({header: props.cardData.header, body: props.cardData.body});
```

Comment:
Again, why not use a spread operator to copy card object? far more easier to handle.